full-service/src/db/transaction_log.rs

```diff
@@ -892,7 +892,12 @@ mod tests {
         let mut ledger_db = get_test_ledger(5, &known_recipients, 12, &mut rng);
 
         // Start sync thread
-        let _sync_thread = SyncThread::start(ledger_db.clone(), wallet_db.clone(), logger.clone());
+        let _sync_thread = SyncThread::start(
+            ledger_db.clone(),
+            wallet_db.clone(),
+            "/webhook",
```

Comment:
⚠️ Changing the signature on `SyncThread::start` to include the URL - so that ultimately the config can launch the thread with the proper URL

Reply:
Went a different direction